vite.plugins.ts

```diff
@@ -0,0 +1,56 @@
+import { assertNonNullish } from "@dfinity/utils";
+import { readFileSync } from "fs";
+
+/**
+ * Read the II canister ID from dfx's local state
+ */
+const readCanisterId = (): string => {
+  const canisterIdsJsonFile = "./.dfx/local/canister_ids.json";
+
+  try {
+    const {
+      internet_identity: { local: canisterId },
+    } = JSON.parse(readFileSync(canisterIdsJsonFile, "utf-8"));
+
+    assertNonNullish(canisterId);
```

Comment:
just checking, this throws right? so we'll get the following message?

> throw Error(`Could not get canister ID from ${canisterIdsJson}: ${e}`);

Reply:
I can add the additional label if you wish, done.